CVE-2016-1897/cvex.yml

```diff
@@ -0,0 +1,10 @@
+blueprint: ubuntu2204-ubuntu2204
+ubuntu1:
+  playbook: ubuntu1.yml
+  command:
+    - "php -S 0.0.0.0:9999 -f /var/www/html"
```

Comment:
This command does not return execution and the whole process hangs. Have you tested your code before committing? If you want to leave this command running in the background, add "&" to the end of the string, i.e. "php -S 0.0.0.0:9999 -f /var/www/html&".